### PR TITLE
Blog: Writing Day project announcement

### PR DESCRIPTION
**Pull request by rosewms**
I was trying to thread the fine line between conversational announcements, plus invitation to continue submissions, and overall enthusiasm. Feedback welcome, I will do my best to apply edits and suggestions ASAP so we can get this queued for tomorrow and attached to the social media.

----
Direct preview link: https://writethedocs-www--2126.org.readthedocs.build/conf/portland/2024/news/announcing-writing-day-projects/

📚 Documentation preview 📚: https://writethedocs-www--2126.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2024/news/announcing-writing-day-projects.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
+:template: {{year}}/generic.html
+:og:image: /_static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg
+
+.. post:: Apr 3, 2024
+   :tags: {{shortcode}}-{{year}}, writing day, projects, oss projects
+
+Meet the first 5 Writing Day 2024 Projects
```
Comment (rosewms) [inline docs/conf/portland/2024/news/announcing-writing-day-projects.rst]: Alt title: Meet the first round of Writing Day Projects

**Comment by goatlady** [comment]
Umm.... I've double checked using fingers AND toes and I'm fairly sure that you've listed 6 projects :)

**Comment by rosewms** [comment]
@goatlady can I have you or someone else make that correction and publish this?

I'm on my to a medical procedure and will be under anesthesia and AFK most of the day.

**Review by plaindocs — APPROVED** [review]
:shipit:

**Pull request merged**